Source/CCRakNetSlidingWindow.cpp

```diff
@@ -218,7 +218,7 @@ void CCRakNetSlidingWindow::OnAck(CCTimeType curTime, CCTimeType rtt, bool hasBA
 		double d = .05;
 		double difference = rtt - estimatedRTT;
 		estimatedRTT = estimatedRTT + d * difference;
-		deviationRtt = deviationRtt + d * (abs(difference) - deviationRtt);
+		deviationRtt = deviationRtt + d * (std::abs(difference) - deviationRtt);

```

Comment:
To avoid problem on other platforms, we could use `fabs` here, and keep `math.h`